mm2src/coins/tendermint/tendermint_balance_events.rs

```diff
@@ -120,7 +121,15 @@ impl EventBehaviour for TendermintCoin {
                             let balance_denom = match self.account_balance_for_denom(&self.account_id, denom).await {
                                 Ok(balance_denom) => balance_denom,
                                 Err(e) => {
-                                    log::error!("{e}");
+                                    log::error!("Failed getting balance for '{ticker}'. Error: {e}");
```

Comment:
I suggest to add `interval` info in log. As user can play with configs,it could be helpful for them to check with what interval the error occurs.

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/blob/fc95ef3ed938a4ff3e85585a306a9299c24bc96e/mm2src/coins/tendermint/tendermint_balance_events.rs#L164-L171

Reply:
> intervals doesn't have any effect for tendermint coins

Ooops, Im sorry, wrong rs file, wanted to mark eth balance code
https://github.com/KomodoPlatform/komodo-defi-framework/blob/dc8bfad8a7fb34f374c4b25864dc245709e2e405/mm2src/coins/eth/eth_balance_events.rs#L105

I suggest to add interval info for eth log
https://github.com/KomodoPlatform/komodo-defi-framework/blob/dc8bfad8a7fb34f374c4b25864dc245709e2e405/mm2src/coins/eth/eth_balance_events.rs#L84